packages/network-explorer/create-peer-id.js

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
```

Comment:
Seems to be a duplicate of `packages/commonwealth/scripts/create-peer-id.js`

Reply:
Good catch, this should have stayed on the other branch.